docs/source/new-docs/getting-started/system-requirements/index.rst

```diff
@@ -3,30 +3,71 @@
 System Requirements
 ===================
 
-Assigned to: Jeremy
-Due for review: June 3rd
+GeoIPS system requirements are strongly dependent on how you intend to use
```

Comment:
No need to use the word strongly here, since these are "requirements" 

(which is already strong phrasing, if we were saying "system hardware preferences", the use of the word strongly here could be helpful in increasing the severity of our recommendation but since we are saving these as requirements, there's no reason to add this modifying word here)

Reply:
I updated this.